server/main/src/main/resources/production.conf

```diff
@@ -0,0 +1,38 @@
+include "main.conf"
+
+akka {
+  loglevel = DEBUG
+
+  remote {
+    netty.tcp {
+      hostname = ${?APP_ADDR}
+      port = 2552
+    }
+  }
+
+  cluster {
+    seed-nodes = [ ${?SEED_NODES} ]
+    jvm-ports = [ ${akka.remote.netty.tcp.port} ]
+    retry = 10s
+    min-nr-of-members = 2
+  }
+
+  persistence {
+    journal.plugin = "cassandra-journal"
+    snapshot-store.plugin = "cassandra-snapshot-store"
+  }
+}
+
+cassandra-journal {
+  contact-points = [ ${?JOURNAL} ]
+}
+
+cassandra-snapshot-store {
+  contact-points = [ ${?SNAPSHOT} ]
+}
+
+kafka {
```

Comment:
`application.conf` includes `main.conf` with an override on the `metadata.broker.list` field. Otherwise `main.conf` has the above values.

I can be more explicit if you think that's a better approach?